src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MemberResolutionKind.cs

```diff
@@ -103,5 +103,10 @@ internal enum MemberResolutionKind : byte
         /// 7.5.3.2 of the language specification).
         /// </summary>
         Worse,
+
+        /// <summary>
+        /// Same as <see cref="Worse"/>, but shouldn't be mentionaed in an ambiguity diagnostics.
+        /// </summary>
```

Comment:
I suggest

"Same as <see cref="Worse"/>, but the candidate shouldn't be mentioned in an ambiguity diagnostics."

(also corrects misspelling of mentioned)